lib/tag-list/style.scss

```diff
@@ -108,138 +107,17 @@ button.tag-list-input {
 }
 
 .icon-reorder {
-  margin-right: 25px;
-}
-
-.editable-list {
-  display: flex;
-  flex-direction: column;
-  overflow: auto;
-  overflow-x: hidden;
-  -webkit-overflow-scrolling: touch;
-  margin: 0;
-  padding: 0;
-  list-style: none;
-
-  .editable-list-item {
-    display: flex;
-    flex: none;
-    padding: 0;
-    overflow: hidden;
-    white-space: nowrap;
-    -webkit-tap-highlight-color: transparent;
-    position: relative;
-    z-index: 10;
-    transition: $anim;
-
-    svg {
-      vertical-align: middle;
-    }
-  }
-
-  &.editable-list-reordering .editable-list-item {
-    transition: none;
-
-    * {
-      cursor: ns-resize;
-    }
-  }
-
-  .editable-list-item-reordering {
-    z-index: 0;
-  }
-
-  .editable-list-item-left {
-    display: flex;
-    align-items: center;
-    flex: 1 0 auto;
-    transition: $anim;
-    transform: translateX(0);
-  }
-
-  &.editable-list-removable .editable-list-item-left {
-    transform: translateX(-24px);
-  }
-
-  .editable-list-item-content {
-    display: flex;
-    flex: 1 1 auto;
-    align-items: center;
-  }
-
-  .editable-list-trash,
-  .editable-list-reorder {
-    cursor: pointer;
-    flex: none;
-    opacity: 0;
-    pointer-events: none;
-    transition: $anim;
-
-    svg {
-      transition: $anim;
-    }
-
-    &:focus {
-      outline: none;
-    }
-  }
-
-  .editable-list-trash {
-    width: 22px;
-    margin-right: 4px;
-
-    svg {
-      fill: $studio-red-40;
-    }
-
-    &:focus svg {
-      fill: darken($studio-red-40, 20%);
-    }
-  }
-
-  .editable-list-reorder {
-    display: flex;
-    align-items: center;
-    flex: none;
-    transform: translateX(24px);
-
-    svg {
-      fill: $studio-gray-50;
-    }
-
-    &:focus svg {
-      fill: $studio-gray-80;
-    }
-  }
-
-  &.editable-list-editing {
-    .editable-list-item-left {
-      transform: translateX(-4px);
-    }
-
-    .editable-list-trash {
-      opacity: 1;
-      pointer-events: auto;
-    }
-
-    .editable-list-reorder {
-      opacity: 1;
-      pointer-events: auto;
-      transform: translateX(0);
-    }
-  }
+  margin-right: 5px;
```

Comment:
Instead of putting these on the icon, let's put them on the parent button. I would recommend giving it its own class (such as `"button-reorder"`) and any color we give it should propagate to its children.

Setting the margin on the icon moves the icon over within its 22x22 parent container, which isn't what we want (and limits how far we can move it). Setting it on the parent container gives us more leeway to pad the icon out, while still keeping it centered within its container.

I hope that makes sense, it's hard to describe.

Here's the button with the parent div padded out.

<img width="332" alt="Screen Shot 2021-01-21 at 7 24 53 PM" src="https://user-images.githubusercontent.com/52152/105441938-6ea4cb00-5c1e-11eb-940e-4ed8a94768f6.png">

Note that we can give it a much bigger margin now if we want:

<img width="340" alt="Screen Shot 2021-01-21 at 7 26 14 PM" src="https://user-images.githubusercontent.com/52152/105442008-8ed48a00-5c1e-11eb-985a-343667f7025e.png">

Also worth noting that this margin, since it is on the parent container, is not clickable.

Here's what happens if we try to set a 100px `margin-right` on the `svg.icon-button` instead. It doesn't have anywhere to go, so it ends up overflowing to the right. Additionally, all of the margin (yellow space) is clickable, because it's within the parent element, so we have a lot of clickable whitespace (sometimes we want this, but probably for icon buttons, we don't).

<img width="266" alt="Screen Shot 2021-01-21 at 7 27 06 PM" src="https://user-images.githubusercontent.com/52152/105442418-af044900-5c1e-11eb-8e96-31b9c146de3a.png">
<img width="339" alt="Screen Shot 2021-01-21 at 7 27 00 PM" src="https://user-images.githubusercontent.com/52152/105442472-b297d000-5c1e-11eb-9d35-6d00cb801b25.png">

Reply:
We might in fact want a slightly bigger margin on this to avoid interfering with the scrollbar.

<img width="120" alt="Screen Shot 2021-01-21 at 6 56 07 PM" src="https://user-images.githubusercontent.com/52152/105443855-96496280-5c21-11eb-895f-45047a83a267.png">

This is 15px:

<img width="103" alt="Screen Shot 2021-01-21 at 7 48 33 PM" src="https://user-images.githubusercontent.com/52152/105443918-b24d0400-5c21-11eb-9d3c-9afe611424eb.png">

Reply:
This makes a lot of sense, thank you for explaining it so well. I've added classes to the buttons and used those.

Based on these and other changes I'm not seeing any overlap with the scrollbar anymore.